explainers/navigation-to-bundles.md

```diff
@@ -0,0 +1,83 @@
+# Explainer: Navigation with Web Bundles<br>(a.k.a. Bundled HTTP Exchanges)
+
+Last updated: Oct 29, 2019
+
+(This explainer was originally written and published as: https://docs.google.com/document/d/1KFmtiE3DHgKfQH5-nKtLiacMrXsoKIXQZ-VIMGHMje0/edit?pli=1#)
```

Comment:
I think we can leave this back-reference in the commit message instead of the body.

Reply:
Good idea, done.